iocage/lib/Jail.py

```diff
@@ -956,6 +982,28 @@ def _configure_route(self, gateway, ipv6=False) -> None:
 
         self.exec(command)
 
+    def require_jail_is_template(self, **kwargs) -> None:
+        """
+        Raise JailIsTemplate exception if the jail is a template
+        """
+        if self.config['template'] is False:
+            raise iocage.lib.errors.JailNotTemplate(
+                jail=self,
+                logger=self.logger,
```

Comment:
wait, why do our exceptions need logger??

Reply:
Their job is to produce log entries. They print with a high loglevel, so that we do not need to care about those error messages in the CLI. We can just exit(1) on IocageException.